src/doc.cpp

```diff
@@ -1556,6 +1560,29 @@ int Doc::PrepareLyricsEnd(FunctorParams *functorParams)
     return FUNCTOR_STOP;
 }
 
+void Doc::SetChildZones() {
+    if (this->GetType() != Facs) return;
+
+    InterfaceComparison ic(INTERFACE_FACSIMILE);
+    ArrayOfObjects children;
+
+    this->FindAllChildByComparison(&children, &ic);
```

Comment:
I usually prefer to have a dedicated Functor for this type of action but we can leave it for now.

Reply:
Wrote `Object::SetChildZones` to be that functor in 449ff5b.